packages/thumbprint-react/components/Modal/index.tsx

```diff
@@ -79,7 +84,7 @@ const ModalAnimatedWrapper = ({
                                 [styles.curtainInnerShouldPageScrollAboveSmall]: shouldPageScrollAboveSmall,
                             })}
                             onClick={shouldCloseOnCurtainClick ? curtainOnClick : undefined}
-                            data-test="thumbprint-modal-curtain"
+                            data-testid="thumbprint-modal-curtain"
```

Comment:
I guess this isn't technically a major version bump since this dataTest prop is not part of the API right now and is only used by Thumbprint's own test suite. Have you done a quick grep in the website to make sure nothing is depending on this anyway. Wouldn't want to do a seemingly safe version bump and break tests.